boards/arm/stm32_min_dev/doc/stm32_min_dev.rst

```diff
@@ -0,0 +1,73 @@
+.. _stm32_min_dev:
+
+STM32 Minimum Development Board
+###############################
+
+Overview
+********
+
+The STM32 Minimum Development Board, is a popular, cheap breadboard friendly
```

Comment:
how about:  ... is a popular and inexpensive breadboard-friendly breakout board ...